packages/contracts/hardhat.config.ts

```diff
@@ -176,7 +176,7 @@ export default <HardhatUserConfig>{
         version: '0.8.9',
         settings: {
           optimizer: {
-            enabled: !isTesting,
+            enabled: true,
```

Comment:
why did we change this? not having the optimizer on while we tests gives us more accurate line numbers on solidity errors